frontend/packages/operator-lifecycle-manager/src/components/operand/utils.ts

```diff
@@ -139,27 +140,75 @@ export const capabilitiesToUISchema = (capabilities: SpecCapability[] = []) => {
   };
 };
 
+const getValidCapabilities = (descriptor: Descriptor<SpecCapability>, jsonSchema: JSONSchema6) => {
+  const schemaType = getSchemaType(jsonSchema ?? {}) as JSONSchemaType;
```

Comment:
Does this work? It's preferable to `as` when possible since it has stronger type checking. I realize we probably don't have typings for react-jsonschema-form.

```suggestion
  const schemaType: JSONSchemaType = getSchemaType(jsonSchema ?? {});
```